packages/ds-healthcare-gov/docs/src/pages/components/Accordion/_Accordion.docs.scss

```diff
@@ -0,0 +1,71 @@
+/*
+Accordion
+
+@uswds https://designsystem.digital.gov/components/accordion/
+
+An accordion is a list of headers that hide or reveal additional content when selected.
+
+Markup: accordion.example.html
+
+@status Draft
+
+Style guide: components.accordion
+*/
+
+/*
+`<Accordion>`
+
+@react-example Accordion.example.jsx
+
+@react-props Accordion.tsx
+
+Style guide: components.accordion.react
+*/
+
+/*
+`<AccordionItem>`
+
+`AccordionItem` renders the Accordion element.
+
+@react-props AccordionItem.tsx
```

Comment:
The docs are showing that the default open and close icons are <AddIcon> and <RemoveIcon>, is there a way to get the react-props here to see that we changed the default props in Accordion/index.tsx?

Reply:
Hm, have made a change and now the default props are not showing up on Hcgov doc site for AccordionItem.